### PR TITLE
simulate.py can update CAN with empty list

### PR DESCRIPTION
**Pull request by SoroushF79**

### CODE REVIEW

File: BSP/Simulator/Hardware/simulate.py [hunk 1]
```diff
@@ -46,8 +46,13 @@ def update_CAN():
     """Periodically update the display state of the CAN bus"""
     global id_text, message_text
     can = CAN.read()
+    
+    if not can: # If can is empty list
+        can.extend(["Empty", "Empty"])
+
```
Comment (JimothyGreene) [inline BSP/Simulator/Hardware/simulate.py:52]: Can you implement this logic in the `CAN.py` file instead? I want the `simulate.py` module to be as unaware as possible as to what's going on behind the scenes.

**Review by JimothyGreene — CHANGES_REQUESTED** [review]
Logic looks good. Just move it to be more specified.

**Review by JimothyGreene — APPROVED** [review]
Looks good and works as far as I can tell

**Review by ErickCortez98 — APPROVED** [review]
Working well on my side too

**Pull request merged**